staging/src/k8s.io/legacy-cloud-providers/aws/aws.go

```diff
@@ -3763,8 +3851,13 @@ func (c *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, apiS
 	}
 
 	if isNLB(annotations) {
-
-		if path, healthCheckNodePort := servicehelpers.GetServiceHealthCheckPathPort(apiService); path != "" {
+		customHealthCheck := func(annotations map[string]string) bool {
```

Comment:
humm, why not just have a variable here instead of this lambda function.
```
_, customProtocolEnabled := annotations[ServiceAnnotationLoadBalancerHealthCheckProtocol]
_, customPortEnabled  := annotations[ServiceAnnotationLoadBalancerHealthCheckPort]
_, customPathEnabled  := annotations[ServiceAnnotationLoadBalancerHealthCheckPath]
customHealthCheckEnabled = customProtocolEnabled || customPortEnabled || customPathEnabled
```

Reply:
refactored the code. no longer needed.